core/src/main/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactory.java

```diff
@@ -759,6 +761,15 @@ static String findDescription(AnnotatedElement annotatedElement) {
         return null;
     }
 
+    /**
+     * Return the path prefix to use. If there is {@link PathPrefix} annotation on the class
+     * then the value from the annotation is used, else falls back to pathPrefix passed through
+     * method pram.
+     */
+    private static String selectPathPrefix(Class<?> clazz, String pathPrefix) {
+        return findFirst(clazz, PathPrefix.class).map(PathPrefix::value).orElse(pathPrefix);
```

Comment:
How about concatenating `pathPrefix` and `PathPrefix.value()`? For example:

```java
ServerBuilder sb = new ServerBuilder();
sb.annotatedService("/foo", new MyService());

@PathPrefix("/bar")
public class MyService { ... }
```

It seems natural to me that `MyService` is bound under `/foo/bar`. What do you think?

Reply:
I agree. Otherwise, I think it can confuse the user. 👍

Reply:
Sounds good. Will make those changes. Thanks for review:)